### DISCUSSION TITLE
Welcome to voronoi Discussions!

### DISCUSSION THREAD

**Discussion by wassfila** [WebSVG/voronoi#20] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ynnebbenny** [comment]
This is exactly what I am looking for. These images below are ai generated lights. I am going to make one with my laser cutter. Drawing the blobs was far more difficult that I imagined. Your generator will help me a LOT.
![366987409_10160438043581590_782335786029169022_n](https://github.com/WebSVG/voronoi/assets/100736840/94092dd9-39c2-45b3-956d-2aae6a5a75fd)
![366336629_10160438043751590_7818388525522396213_n](https://github.com/WebSVG/voronoi/assets/100736840/ac4094cc-1881-4d16-af7a-b75980a42dfa)
![366822464_10160438043491590_4497308113573653483_n (1)](https://github.com/WebSVG/voronoi/assets/100736840/f63df27c-00dd-4997-860f-b487db7fc5af)
![366936748_10160438043211590_2655512897177945071_n](https://github.com/WebSVG/voronoi/assets/100736840/fc1afc8b-a0f1-4210-a9c0-c62f29712fb8)
![366939296_10160438043441590_1212184606791481248_n](https://github.com/WebSVG/voronoi/assets/100736840/a1e13d0d-48ac-412b-b5e3-43ab9df3901b)
![366947094_10160438043126590_198953223574500084_n](https://github.com/WebSVG/voronoi/assets/100736840/84465052-3e12-4fbf-befb-27df11319b50)
![366954451_10160438043926590_151426538655359426_n](https://github.com/WebSVG/voronoi/assets/100736840/6ba534f5-9b71-4191-b07d-353da583afe6)
![366979451_10160438043156590_653090706643031023_n](https://github.com/WebSVG/voronoi/assets/100736840/47bc6401-f59f-4113-8752-7cc47c08b2a6)
![366979626_10160438043376590_4126109055049444551_n](https://github.com/WebSVG/voronoi/assets/100736840/ff357142-fdf9-48a0-8e26-24e4f8be9efb)

**Reply by wassfila** [reply]
Awesome, thanks for the feedback.
